content/docs/common-areas/backups.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Backups
+weight: 4
+description: Taking regular backups of your website can help you in effectively managing the downtime situation.
+---
+
+# Backups
+
+The saying - "Always have a backup plan" stands true and is an integral part of web application management too. Although a lot of measures can be taken to avoid the downtime of a website but it is hard to predict whether we will ever come across this situation or not. We should be well prepared for anything. Hence, having a back up plan or in other words, taking regular backups of your website can help you in effectively managing the downtime situation. The website backups allows you to back up your site and related database via FTP, SFTP, FTPS, or SSH key to a secure cloud location.
```

Comment:
Should we also recommend taking backups as part of the process before certain tasks, like deployments, running database updates, migrations, module/core updates, etc?

I think calling out such situations might help.

Reply:
True @swarad07 !
Thanks for the suggestion here. I have added a separate section to highlight these situations.

Please review and let me know if anything else needs to be added here.

Thanks.